### PR TITLE
Add bob.ip.facedetect recipe

### PR DESCRIPTION
**Pull request by 183amir**

### CODE REVIEW

File: recipes/bob.ip.facedetect/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    - bob
+    - bob.ip
+    - bob.ip.facedetect
+
+  requires:
+    - nose
+
+about:
+  home: http://github.com/bioidiap/bob.ip.facedetect
+  license: Modified BSD License (3-clause)
```
Comment (msarahan) [inline recipes/bob.ip.facedetect/meta.yaml]: We have been shortening this to 3-clause BSD.  Would you like to modify it, or would you prefer not to?

**Comment by msarahan** [comment]
Quick comment about license name - then I think this is ready for merge.

File: recipes/bob.ip.facedetect/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/bob.ip.facedetect/meta.yaml]: We have been allowing this one to slide actually. We were really strict for a little while, but there were some disagreements as to what qualified as an acceptable license string. This one has been allowed in practice. Though we should really pick some strict criteria and stick with them.

**Comment by jakirkham** [comment]
Thanks @183amir.

**Pull request merged**